py3-ply.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: py3-ply
+  version: 3.11
+  epoch: 0
+  description: Python Lex & Yacc
+  copyright:
+    - license: BSD
+  dependencies:
+    runtime:
+      - python3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python3
+      - py3-setuptools
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/dabeaz/ply
+      tag: ${{package.version}}
+      expected-commit: 0f398b72618c1564d71f7dc0558e6722b241875a
+
+  - name: Python Build
+    runs: python setup.py build
+
+  - name: Python Install
+    runs: python setup.py install --prefix=/usr --root="${{targets.destdir}}"
+
+  - uses: strip
+
+# some sort of a deprecated project
+update:
+  enabled: false
```

Comment:
this and [related noticed](https://github.com/dabeaz/ply#important-notice---october-27-2022) may be a problem as we can only add maintained projects to wolfi

Reply:
it might be yes but some packages depend on it "py3-pythran" to be precise

Reply:
We can package it, but we should indicate the date of the git snapshots we use.

Reply:
thanks, I've updated it as `3.11_git20180215`